sql/src/test/java/io/crate/planner/SelectPlannerTest.java

```diff
@@ -445,13 +445,6 @@ public void testSelectAnalyzedReferenceInFunctionAggregation() throws Exception
         e.plan("select min(substr(text, 0, 2)) from users");
     }
 
-    @Test
-    public void testSelectNonIndexedReferenceInFunctionAggregation() throws Exception {
-        expectedException.expect(IllegalArgumentException.class);
-        expectedException.expectMessage("Cannot select non-indexed column 'no_index' within grouping or aggregations");
-        e.plan("select min(substr(no_index, 0, 2)) from users");
```

Comment:
Why does this not work anymore? Or is this tested elsewhere?

Reply:
this was an artificial limitation as aggregations on non-indexed columns always worked by using the column store (doc_values)